kymatio/scattering2d/backend/backend_torch.py

```diff
@@ -27,13 +27,10 @@ def __init__(self, pad_size, pre_pad=False):
         self.padding_module = ReflectionPad2d(pad_size)
 
     def __call__(self, input):
-        if(self.pre_pad):
-            output = input.new_zeros(input.size(0), input.size(1), input.size(2), input.size(3), 2)
-            output.narrow(output.ndimension()-1, 0, 1)[:] = input
-        else:
-            out_ = self.padding_module(input)
-            output = input.new_zeros(*(out_.size() + (2,)))
-            output.select(4, 0)[:] = out_
+        if(not self.pre_pad):
+            input = self.padding_module(input)
+        output = input.new_zeros(*(input.size() + (2,)))
+        output.select(4, 0)[:] = input
```

Comment:
what does this do? Can you make it less torch-obscure and more numpy-like?

Reply:
if s is the size, it creates an array of size (s,2) to handle complex number and set the imaginary part to 0.. not sure how to do make it more numpy-like, I'm not familiar with the standards